src/templates/macros/task_macros.html.twig

```diff
@@ -0,0 +1,47 @@
+{% macro taskListingCurrent(tasksCurrent) %}
+<section class="listing current_week content-flex-item">
+<h2>Current week</h2>
+<table class="tasks-table">
+{% for task in tasksCurrent %}
+<tr>
```

Comment:
The indentation is broken - it doesn't affect functionality, but it's hard to read and easy to make a mistake